packages/eas-cli/src/project/remoteVersionSource.ts

```diff
@@ -98,3 +123,88 @@ export function getBuildVersionName(platform: Platform): string {
     return 'buildNumber';
   }
 }
+
+export async function ensureAppVersionSourceIsSetAsync(
+  easJsonAccessor: EasJsonAccessor,
+  easJsonCliConfig: EasJson['cli'] | undefined,
+  nonInteractive: boolean
+): Promise<EasJson['cli'] | undefined> {
+  let selectOption;
+  if (nonInteractive) {
+    Log.warn(
+      '"appVersionSource is not specified. The previous default value for "appVersionSource" used to be "local", but it has been changed to "remote".'
```

Comment:
```suggestion
      '"appVersionSource" is not specified. The previous default value for "appVersionSource" used to be "local", but it has been changed to "remote".'
```

Let's add that missing end quote on the first word.